src/main/webapp/map.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Hello Google Maps</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBZkWQmagGmCHKRX2osG0BSWiUYLOZaxWA"></script>
+    <script>
+        function createMap(){
```

Comment:
Consider putting this javascript code in a separate file if possible!

Reply:
we should try to do that from now on. Im going through past features i have done and am transferring javascript code to separate files :)

Reply:
Sounds Good